packages/pkg-tests/pkg-tests-fixtures/packages/no-deps-scripted-1.0.0/package.json

```diff
@@ -2,8 +2,8 @@
     "name": "no-deps-scripted",
     "version": "1.0.0",
     "scripts": {
-        "preinstall": "echo 'module.exports.push(100);' >> log.js",
-        "install": "echo 'module.exports.push(200);' >> log.js",
-        "postinstall": "echo 'module.exports.push(300);' >> log.js; echo 'module.exports = '\"$(node -p 'Math.floor(Math.random() * 512000)')\"';' > rnd.js"
+        "preinstall": ":; echo 'module.exports.push(100);' >> log.js; exit $?\necho module.exports.push(100); >> log.js",
```

Comment:
@arcanis This doesn't work in Windows. In a regular batch file would work, but on these inlined scripts, everything after `\n` is ignored. I tested:
		"hola": "\necho hola",
And then `yarn run hola` prints outputs nothing. Just removing the `\n` makes it work.
I'm still trying to figure a workaround for these tests that works on every OS.

Reply:
I believe removing the quotes and the semicolon it would work across OS.
Except for the 3rd `postinstall` , where we generate a `rnd.js` file with a `$(...)` syntax that doesn't seem to have an equivalent in Windows. Instead, we could use `%RANDOM%`. Let me try...